CHANGELOG.md

```diff
@@ -17,6 +25,103 @@
+- (Contrib) `architecture`: New Tier 3 platform riscv64 allowing the collector to be built and distributed for this platform. ([#41507](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41507))
+- (Contrib) `redactionprocessor`: The new `redact_all_types` option redacts all field types after casting them to string. ([#36684](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36684))
+- (Contrib) `sqlserverreceiver`: Add 'sqlserver.computer.uptime' metric ([#41207](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41207))
+- (Contrib) `sqlserverreceiver`: Make queries compatible with Azure SQL Database and Azure SQL Managed Instance. ([#41102](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41102))
+- (Contrib) `telemetrygen`: Adds a new telemetrygen flag to generate unique timestamps for within a second interval ([#39933](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39933))
+
+### 🧰 Bug fixes 🧰
+
+- (Core) `exporterhelper`: Prevents the exporter for being stuck when telemetry data is bigger than batch.max_size ([#12893](https://github.com/open-telemetry/opentelemetry-collector/issues/12893))
+- (Core) `mdatagen`: Fix import paths for mdatagen component ([#13069](https://github.com/open-telemetry/opentelemetry-collector/issues/13069))
+- (Core) `otlpreceiver`: Error handler correctly fallbacks to content type ([#13414](https://github.com/open-telemetry/opentelemetry-collector/pull/13414))
+- (Core) `pdata/pprofiles`: Fix profiles JSON unmarshal logic for originalPayload. The bytes have to be base64 encoded. ([#13483](https://github.com/open-telemetry/opentelemetry-collector/pull/13483))
+- (Core) `xpdata`: Fix unmarshaling JSON for entities, add e2e tests to avoid this in the future. ([#13480](https://github.com/open-telemetry/opentelemetry-collector/pull/13480))
+- (Core) `service`: Downgrade dependency of prometheus exporter in OTel Go SDK ([#13429](https://github.com/open-telemetry/opentelemetry-collector/pull/13429))
+  This fixes the bug where collector's internal metrics are emitted with an unexpected suffix in their names when users configure the service::telemetry::metrics::readers with Prometheus
+- (Core) `service`: Revert Default internal metrics config now enables `otel_scope_` labels ([#12939](https://github.com/open-telemetry/opentelemetry-collector/issues/12939), [#13344](https://github.com/open-telemetry/opentelemetry-collector/pull/13344))
+  Reverting change temporarily due to prometheus exporter downgrade. This unfortunately re-introduces the bug that instrumentation scope attributes cause errors in Prometheus exporter. See http://github.com/open-telemetry/opentelemetry-collector/issues/12939 for details.
```

Comment:
These two shouldn't have been added to the core changelog. They were reverted before the release. Looks like a bad merge of a patch branch. I'll submit a PR upstream to remove those lines from the changelog